mender-convert-functions.sh

```diff
@@ -1,12 +1,14 @@
 #!/bin/bash
 
-# Partition alignment value in bytes (8MB).
-partition_alignment=8388608
+# Partition alignment value in bytes (4MB).
+partition_alignment=4194304
 # Boot partition storage offset in bytes (equal to alignment).
 vfat_storage_offset_regular=$partition_alignment
```

Comment:
Also here we have differentiate between BBB & RPI3. Regular is for BBB and should be 8MB, extended should be for RPI3 and equal to 12MB.